src/Core/src/Fonts/FontRegistrar.Windows.cs

```diff
@@ -1,13 +1,23 @@
 #nullable enable
 using System.IO;
+using System.Reflection;
 
 namespace Microsoft.Maui
 {
 	public partial class FontRegistrar : IFontRegistrar
 	{
 		string? LoadNativeAppFont(string font, string filename, string? alias)
 		{
-			var root = global::Windows.ApplicationModel.Package.Current.InstalledLocation.Path;
+			string root;
+			try
+			{
+				root = global::Windows.ApplicationModel.Package.Current.InstalledLocation.Path;
```

Comment:
Is there a non-exceptional way to do this? Otherwise unpackaged apps will all get exceptions on startup?